core/templates/dev/head/pages/exploration_editor/editor_tab/ResponsesService.js

```diff
@@ -107,7 +107,12 @@ oppia.factory('ResponsesService', [
       if (updates.dest) {
         answerGroup.outcome.dest = updates.dest;
       }
-      answerGroup.correct = false;
+      if (updates.hasOwnProperty('labelledAsCorrect')) {
+        answerGroup.labelledAsCorrect = updates.labelledAsCorrect;
+      }
+      else {  
```

Comment:
Drop this clause altogether?

Reply:
Sorry, I didn't get it. Please elaborate.

Reply:
If the `updates` dict doesn't have the property, then that says nothing about what to set labelledAsCorrect to (look at the rest of the clauses in this method). So I don't think you should have the else clause here (i.e. treat nonexistence of the property as false). Right?

Reply:
Yep! Thanks! I will remove the else clause.

Reply:
Done!